pyquil/paulis.py

```diff
@@ -82,7 +83,10 @@ def id(self):
             return s
 
     def __eq__(self, other):
-        return self.id() == other.id()
+        if not isinstance(other, PauliTerm):
+            return False
+        print self.coefficient, other.coefficient
```

Comment:
please no `print`'s in this code. This will blow up the output for large circuits. Consider adding a logger, but that might be out of scope

Reply:
@jotterbach Yep, it wasn't supposed to be there- removed and the tests are running again.